src/components/VCarousel/VCarousel.vue

```diff
@@ -71,6 +62,11 @@
 
         !this.isBooted && this.cycle && this.restartInterval()
         this.isBooted = true
+
+        this.$emit('input', this.current)
+      },
+      value () {
```

Comment:
You haven't defined `value` as a prop. I would probably try to remove `current` entirely in favor of `value` and emitting `input`.